plugins/inputs/t128_graphql/t128_graphql.go

```diff
@@ -16,9 +16,12 @@ import (
 	"github.com/Jeffail/gabs"
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
+	"github.com/influxdata/telegraf/metric"
 	"github.com/influxdata/telegraf/plugins/inputs"
 )
 
+var T128GraphQLMetrics []telegraf.Metric
```

Comment:
Besides global vars usually being not the answer. How does it work if there are multiple inputs trying to use this? They would all go into the same slice and then the output would iterate. The input plugin structs are instantiated once per input in the telegraf config, but this global will only have one instance for the whole telegraf instance.

Reply:
There is no mechanism for direct communication. between telegraf because each plugin operates independently and there is no direct way to access metrics generated by another input. I do agree that a global variable is not the way. I can use an intermediate output plugin to access the data! I believe that can help solve a lot of problems